src/vt-tv/render/render.h

```diff
@@ -67,9 +67,61 @@
 #include <vtkThresholdPoints.h>
 #include <vtkWindowToImageFilter.h>
 #include <vtkPNGWriter.h>
+#include <vtkRegularPolygonSource.h>
+#include <vtkSphereSource.h>
+#include <vtkBitArray.h>
+
+#include <vtkPolyDataWriter.h>
+#include <vtkExodusIIWriter.h>
+#include <vtkXMLPolyDataWriter.h>
+
+#include "vt-tv/api/rank.h"
+#include "vt-tv/api/info.h"
+
+#include <fmt-vt/format.h>
+#include <ostream>
+#include <cmath>
+#include <algorithm>
+#include <iterator>
+#include <cstdlib>
+#include <tuple>
+#include <limits>
+#include <map>
+#include <unordered_set>
+#include <set>
 
 namespace vt { namespace tv {
 
+template <typename T>
```

Comment:
This would probably more efficiently and cleanly implemented with std::array and just forward the `[]` operator.

Reply:
In fact, given how it's used, you might just want to delete this and replace with std::array.

Reply:
I replaced it entirely with `std::array<typename, 3>`, and replaced the vectors I was using to store 3 elements with it as well.